package.json

```diff
@@ -62,14 +62,9 @@
     "eslint": "8.17.0",
     "eslint-config-standard": "17.0.0",
     "eslint-config-standard-jsx": "11.0.0",
-    "eslint-plugin-import": "2.26.0",
```

Comment:
same here: activate or remove? or is it unnecessary?

Reply:
this might be helpful (removed because we didnt activate it, but will do).